test/e2e/runner/setup.go

```diff
@@ -324,7 +324,6 @@ func MakeConfig(node *e2e.Node) (*config.Config, error) {
 		if len(cfg.StateSync.RPCServers) < 2 {
 			return nil, errors.New("unable to find 2 suitable state sync RPC servers")
 		}
-		cfg.StateSync.MaxDiscoveryTime = 30 * time.Second
```

Comment:
Are we sure that by removing this we are testing all possible corner cases of statesync now?

Reply:
Do you know why it was introduced in the first place?